app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java

```diff
@@ -207,7 +217,18 @@ private void loadFragment(int navItemId) {
                 fragment = EventDashboardFragment.newInstance(eventId);
                 break;
         }
-        fragmentManager.beginTransaction().replace(R.id.fragment_container, fragment).commit();
+        if(fragmentManager.getBackStackEntryCount()==0){
```

Comment:
When we have control over pushing and popping the state, we should know that what state we are in and not need to get the backstack entry count. This is bad because it means we have limited the capability of the stack to a constant where this is not the desired behaviour.